webcomponents/custom.html

```diff
@@ -51,25 +51,43 @@
                 "keypress" : "handleKeyPress"
             },
             toggleLongDesc: function(e) {
+                var curName = $(this).attr("dg-name");
+                var $context;
+                var handleFocus = true;
+                if(curName === "*"){
+                    //All elements on the page.
+                    handleFocus=false;
+                    //Don't use document, because dilter doesn't actually do what you'd expect.
```

Comment:
I can make this more clear, basically, $(document).filter doesn't find 
the item, because it's not a list, while $("[dg-name=bla]") generates a 
list of all gd-names with name of x


On 7/6/2016 11:57 AM, Charles LaPierre wrote:
>
> In webcomponents/custom.html 
> <https://github.com/benetech/dgmr-rdm-web/pull/2#discussion_r69779596>:
>
> > @@ -51,25 +51,43 @@
> >                  "keypress" : "handleKeyPress"
> >              },
> >              toggleLongDesc: function(e) {
> > +                var curName = $(this).attr("dg-name");
> > +                var $context;
> > +                var handleFocus = true;
> > +                if(curName === "*"){
> > +                    //All elements on the page.
> > +                    handleFocus=false;
> > +                    //Don't use document, because dilter doesn't actually do what you'd expect.
>
> comment "dilter"?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/benetech/dgmr-rdm-web/pull/2/files/7c54953b7c6bb7cf3fc15ec8192d07dc908402e5..e869d1b138bcc26becf49507d25b69c56e0e8e26#r69779596>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/AFGivdq9jRIFG13nm42Fw3tL3lVsfA4bks5qS-ySgaJpZM4JGVBf>.
>

-- 
------------------------------------------------------------------------


    Derek Riemer

  * Department of computer science, third year undergraduate student.
  * Proud user of the NVDA screen reader.
  * Open source enthusiast.
  * Member of Bridge Cu
  * Avid skiier.

Websites:
Honors portfolio <http://derekriemer.com>
Awesome little hand built weather app! 
<http://django.derekriemer.com/weather/>

email me at derek.riemer@colorado.edu <mailto:derek.riemer@colorado.edu>
Phone: (303) 906-2194